src/tools/remote-test-client/tests/lib.rs

```diff
@@ -8,3 +8,18 @@ fn test_help() {
     let stdout = String::from_utf8(output.stdout.clone()).unwrap();
     assert!(stdout.trim().starts_with("Usage:"));
 }
+
+#[test]
+fn test_timeout() {
+    let mut cmd = assert_cmd::cargo::cargo_bin_cmd!();
+    cmd.env("TEST_DEVICE_CONNECT_TIMEOUT_SECONDS", "1");
+    cmd.env("TEST_DEVICE_ADDR", "0.0.0.0:6969");
+    cmd.args(["spawn-emulator", "riscv64-unknown-linux-gnu", "./"]);
```

Comment:
let's emphasize more that we are not _really_ interested in spawning an emulator


```suggestion
    cmd.args(["spawn-emulator", "dummy-target", "dummy-server", "dummy-tmpdir"]);
```